source/style-guide/style/capitalization.txt

```diff
@@ -46,48 +54,38 @@ capitalization. Always follow the capitalization that's used by the
 company, shown in a :mw:`dictionary </>`, or accepted as standard in
 the industry.
 
-.. list-table::
-   :widths: 50 50
-   :header-rows: 1
-
-   * - Examples
-     -
-
-   * - MongoDB
-     - Service Advertising Protocol
-
-   * - Hong Kong
-     - WordPress
-
-   * - fantastic experience
-     - RackConnect
-
-   * - Fanatical Support
-     - Boolean
-
-   * - Cloud Servers
-     - OpenStack
-
-   * - Linux
-     - Internet
-
-   * - Microsoft Windows
-     - Ethernet
-
-   * - SQL Server
-     - Wi-Fi
-
-   * - PuTTY
-     - lighttpd
-
-For the correct capitalization of MongoDB product names, see the
-:website:`MongoDB corporate website </>`.
-
-For the correct capitalization of some commonly used third-party names,
-see :ref:`third-party-names-and-trademarks`.
+.. hlist::
+   :columns: 4
+
+   - Boolean
+   - Cloud Servers
+   - Ethernet
+   - Fanatical Support
+   - fantastic experience
+   - Hong Kong
+   - Internet
+   - lighttpd
+   - Linux
+   - Microsoft Windows
+   - MongoDB
+   - OpenStack
+   - PuTTY
+   - RackConnect
+   - Service Advertising Protocol
+   - SQL Server
+   - Wi-Fi
+   - WordPress
+
```

Comment:
Ready for more fun? Context-sensitive capitalization:

- "FTP" - the protocol, introduced [here](https://tools.ietf.org/html/rfc354) (and check out the gnarly capitalization of HOSTs on that RFC!)
- `ftp` - the command line FTP client, included on every *nix and macOS box.

Reply:
Also stellar inclusion of `PuTTY`, which highlights that there are no rules when it comes to naming your software (it's a play on the shell's TTY interface, but it totally loses its charm -- and ability to stand out in a crowd -- if just "Putty").